.github/workflows/build.yml

```diff
@@ -50,11 +50,11 @@ jobs:
     - name: mkdir
       run: mkdir -p out
     - name: cmake
-      run: cmake .. -G Ninja -DWERROR=ON
+      run: cmake .. -G Ninja -DWERROR=ON -Werror=dev
```

Comment:
what is -Werror=dev?

Reply:
This is cmake thing that makes cmake warnings into errors..  nothing to do with C/C++.  Helps us avoid ugly cmake warnings.